drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -209,6 +209,7 @@
 
 .xblock--drag-and-drop .zone {
     position: absolute;
+    border: 1px hidden #f00;
```

Comment:
@agaylard I'm not sure about using `#f00` (red) for the border color. I would go for something a little more "neutral", i.e., a shade of `#666` (which is currently used for borders in the Studio editor). For accessibility reasons, we do want good contrast relative to the default image. Something like `#565656` or even `#373737` would work. When combined with the color of the darkest portion of the background image (`#c9db38`), the former passes WCAG AA for normal text, and the latter passes WCAG AAA for normal text. (I used [this tool](http://webaim.org/resources/contrastchecker/) to check contrast values.)

When you change the color here, please adjust the color used for borders in the Studio editor as well.